request.schema.json

```diff
@@ -12,6 +12,105 @@
             "type": "string",
             "pattern": "^data:image/[^;]+;base64,[a-zA-Z0-9+/=]+$"
         },
+        "highChartsData": {
+            "description":"Data extracted from the highchart",
+            "type": "object",
+            "properties":{
+                "title":{
+                    "type":"string",
+                    "description": "Title of the graph"
+                },
+                "subtitle":{
+                    "type":"string",
+                    "description":"Sub title of the graph"
+                },
+                "type":{
+                    "type": "string",
+                    "description": "Type of graph (line/pie/bar)"
+                },
+                "xAxis":{
+                    "description":"Information about x-axis of the graph",
+                    "type": "object",
+                    "properties": {
+                        "labels":{
+                            "description": "Labels present on the x-axis",
+                            "type" : "array",
+                            "items": {
+                                "type":"string"
+                            }
+                        },
+                        "description":{
+                            "description": "Description text about x-axis",
+                            "type": "string"
+                        }
+                    }
+                },
+                "yAxis":{
+                    "description":"Information about y-axis of the graph",
+                    "type": "object",
+                    "properties": {
+                        "description":{
+                            "description": "Description text about y-axis",
+                            "type": "string"
+                        }
+                    }
+                },
+                "noOfDataPoints": {
+                    "type": "number",
+                    "description": "Maximum number of data points present in all the series in the graph"
+                },
+                "data":{
+                    "description": "Collection of all the series in the graph",
```

Comment:
This only includes the series property. Is this a simplification of the original format?